TelemetrySite/server/data_upload_scripts/data_upload.py

```diff
@@ -45,14 +48,37 @@ def submit_data(mf4_file, dbc_file, context_data):
     
     context_data["event"]["runs"][0]["mf4File"]=fs.put(mf4_file, encoding="utf-8")
     context_data["event"]["runs"][0]["dbcFile"]=fs.put(dbc_file, encoding="utf-8")
-    # overall upload can not exceed 16 mb
-    # splice data until it is 16 mb then submit it
-    # add data as needed
-    object_id = collection_access_events.insert_one(context_data).inserted_id
+
+    return(upload_data_in_chunks(context_data, data_values_json))
+
+
+def add_data(mf4_file, dbc_file, new_run_data, mongo_doc_id):
+    db_connection = create_db_connection()
+    fs = gridfs.GridFS(db_connection)
+
+    dbc_decoded = cantools.database.load_file(dbc_file)
+
+    # get a dictionary of CAN id -> Board name
+    can_id_values = get_board_names(dbc_decoded)
+    # create an outline of how to read the data
+    config_values = createConfig(can_id_values, dbc_decoded)
+    # turn data from CAN messages -> list
+    data_values_json = parse_data(mf4_file, config_values)
+
+    # get just the needed portion
+    new_run_data = json.loads(new_run_data)["event"]["runs"][0]
```

Comment:
Update this code. It gets rid of the event data we need for the rest of the runs. This is an outdated piece of code from when we grouped all runs under one event documents
```suggestion
    new_run_data = json.loads(new_run_data)
```